python_modules/jupedsim/jupedsim/sqlite_serialization.py

```diff
@@ -114,12 +125,17 @@ def begin_writing(self, simulation: Simulation) -> None:
         except sqlite3.Error as e:
             cur.execute("ROLLBACK")
```

Comment:
We can no longer do this if journal mode is disabled, we can just remove this line.